include/custom_op/kernel_context.h

```diff
@@ -29,6 +30,7 @@ class CUDAKernelContext :  public KernelContext {
   virtual void* GetCudaStream() const = 0;
   virtual void* GetCublasHandle() const = 0;
   virtual int GetCudaDeviceId() const = 0;
+  virtual void* GetScratchBufferUnderMultiStream(const OrtMemoryInfo* , size_t ) { return nullptr; }
```

Comment:
I can you cannot use ORT type here since these classes will be used in eager mode which doesn't depend on ORT

Reply:
yes, we need to decouple it from ORT types.